src/site/labs/hello-world/hello-world.rst

```diff
@@ -72,7 +72,7 @@ Kattis
 
     I realize many of you will have solved these already in Python. Good, but now do it in Java.
 
-    Don't know how to do things in Java? Check out :doc:`topic 1 </topic1>` and :doc:`topic 2 </topic2>`.
+    Don't know how to do things in Java? Check out :doc:`topic 1 </topics/intro/introduction>` and :doc:`topic 2 </topics/java-vs-python/java-vs-python>`.
```

Comment:
I'd change things like _topic n_ here to the topic name. If you re-order topics in the future, it'd be impossible to track all these usages down. It's worse than that in general, removing a topic, for example, requires changing of potentially **all** references, not just ones to the removed topic (as topic n becomes topic n-1).

Fine as a follow up/new issue, I don't think it's really related to #46, as the same problem exists in the previous directory layout.

Reply:
Damn dude, good eye. It was only the actual references I thought of. That's some grade A sleuthing.